### ISSUE TITLE
Problem with the extension sync

### ISSUE THREAD

**Issue by 00jackson** [QasimWani/LeetHub#516]
[##]()) Please review previous closed issues before filling out a new one! Duplicate issues will be closed without comment.

**Describe the bug**
I think even after syncing my acc with leethub its not connected with my leetcode acc and it is not pushing my codes.
**To Reproduce**

**Expected behavior**
How does it work cause once I code it should push my codes instantly but it isn't.

**Screenshots**
i have attached the screenshots of the recent commit.

<img width="932" alt="Screenshot 2023-03-19 at 6 18 59 PM" src="https://user-images.githubusercontent.com/95626883/226176501-8fb79cfa-f71f-48d0-b0bb-b027720657a4.png">
<img width="724" alt="Screenshot 2023-03-19 at 6 18 25 PM" src="https://user-images.githubusercontent.com/95626883/226176502-6c3f6fb3-e4f3-40e9-8d45-082a63d50a3b.png">

**Comment by veerviren** [comment]
Yes I also face the same issue accepted problem is not reflected in GitHub

**Comment by ani280919** [comment]
Did you try reverting back to old version of leetcode? 
For me, it worked once, and now again not working!

**Comment by randomDev04** [comment]
I had the same issue, But after reverting to the previous version, it synced the Problems.

**Comment by soorya97** [comment]
+1
Its broken with the new version, 
With the old version - Its working fine

**Comment by randomDev04** [comment]
In my case, it Shows Some 422 error in the network call

**Comment by dannutiee** [comment]
+1 
luckily found this thread 😻  back to the old version solved the problem for me too, I hope that they won't switch it off permanently soon 😨

**Comment by Suyash1463** [comment]
How to switch to old version??

**Comment by miker704** [comment]
Hi, I ve been experiencing the same issue apparently leetcode has made some changes to their site mainly their U.I. if your using the new version of the site, completing and submitting a problems solution that passes successfully will not push to github nor will the problem solved counter will increase. To fix this you must switch to the old version of leetcode and re-submit the problem for it to sync to github and increase your problem solved counter. The LeetHub extension will have to make an update to work with both the new leetcode layout in addition to the old version. In the meantime you will have to use the old version of leetcode or switch to the old version to submit the solution in order for it to sync. To do this click on your profile pic/account on the upper right side of leetcode to open the drop down menu and click the second to last option "Revert to old version" the site will refresh to the normal white version of the site and will be able to sync to github the next time you submit a successful solution. you can revert to the new version again anytime by clicking the new button that lingers around the middle right side of the text editor. You can also go to your profile dashboard and click the button on the lower right hand side that is above a blue smiley bubble to switch to the old version. if your in the old version already there will be a button called new to switch to the new version in the same location

**Comment by miker704** [comment]
> How to switch to old version??

refer to the comment above i just posted

**Comment by miker704** [comment]
![image](https://user-images.githubusercontent.com/33719996/230086555-88730ee8-a605-42ed-b53f-aa9db1a74dc4.png)

the button i drew a circle around is what you need to press to switch to the old version if your in your account dashboard

**Comment by 00jackson** [comment]
Ok thanks :)

**Comment by Suyash1463** [comment]
@miker704 Thank you for the answer!
